src/xbot/common/math/PIDManagerFactory.java

```diff
@@ -4,7 +4,7 @@
 
 public interface PIDManagerFactory {
```

Comment:
I'm kiiiiiind of leaning towards `PIDFactory` if it makes a bunch of different PID-related things? No strong feelings though.